scripts/circleci_auto_approve.py

```
@@ -0,0 +1,173 @@
+
+def is_org_member(config: Config) -> bool:
+  if config.pr_author in config.trusted_users:
+    print(f"{config.pr_author} is explicitly trusted")
+    return True
+
+  token = os.environ.get("GITHUB_ORG_READ_TOKEN") or config.github_token
+  url = f"{GITHUB_API}/orgs/{config.trusted_org}/members/{config.pr_author}"
+  try:
+    request_json(url, token=token)
+    print(f"{config.pr_author} is a member of {config.trusted_org}")
+    return True
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)